Source/WebCore/platform/graphics/NullGraphicsContext.h

```diff
@@ -123,7 +123,6 @@ class NullGraphicsContext : public GraphicsContext {
     void drawFocusRing(const Vector<FloatRect>&, float, float, const Color&) final { }
 
     void drawImageBuffer(ImageBuffer&, const FloatRect&, const FloatRect&, const ImagePaintingOptions& = { }) final { }
-    void drawConsumingImageBuffer(RefPtr<ImageBuffer>, const FloatRect&, const FloatRect&, const ImagePaintingOptions& = { }) final;
```

Comment:
weird, where's the implementation of this :)

Reply:
The implementation of it is in GraphicsContext.cpp. I think it was put in a source file instead of this header because the `RefPtr<ImageBuffer>` is passed by value which requires including ImageBuffer.h.